examples/scalajs-react/index.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html lang="en" data-framework="scalajs-react">
+	<head>
+		<meta charset="utf-8">
+		<title>scalajs-react • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<section id="todoapp" class="todoapp"></section>
```

Comment:
yeah, good point - it was left over and i used it to find where in the dom i would render, changed it to lookup by class instead